docs/quick-starts/rug-typescript.md

```diff
@@ -47,7 +47,7 @@ Once you have these two installed on your machine you should be able to successf
 
 ```shell
 $ npm install
-@russmiles/scattered-rugs@0.8.0 ...scattered-rugs/.atomist
+@yourusername/scattered-rugs@0.8.0 ...scattered-rugs/.atomist
 └── @atomist/rug@0.10.0
```

Comment:
Perhaps default version in sample should be 0.1.0 to agree with other content and be less surprising.